src/app/(auth)/login/page.tsx

```diff
@@ -49,26 +35,24 @@ export default function Login() {
           <TextInput
             label="Email"
             placeholder="email@example.com"
-            erroring={false}
-            errorText="Email Error" // {errorMessage}
+            errorText="" // "Email Error"
             type="email"
-            name="email"
+            id="email"
             value={email}
             setValue={setEmail}
           />
           <TextInput
             label="Password"
             placeholder="Password"
-            erroring={false}
-            errorText="Password Error"
+            errorText="" // "Password Error"
             type="password"
-            name="password"
+            id="password"
             value={password}
             setValue={setPassword}
           />
         </QuestionsDiv>
         <P>
-          <AColored $color={COLORS.greyMid} href="/reset-password">
+          <AColored $color={COLORS.greyMid} href="/forgot-password">
```

Comment:
Not sure if this is just on my end, but every time I click this for the first time, it renders the `/login` route again 🤔. It's only on the second try that it actually renders `/forgot-password`....

@jinkang-0 curious if you have thoughts on why this might be?

Reply:
I'm not sure if I'm having the same issue...

Reply:
agh ok it might just be on my end! @ccatherinetan if it looks fine to you too, think we're good :+1:

Reply:
I will add that it may be better if we use the `Link` component from `next/link` to navigate so the page transition is more seamless. More [here](https://nextjs.org/docs/app/api-reference/components/link).

Reply:
gotcha, i'll implement Link! but otherwise the routing is working fine for me too.

should i use Link for rerouting between other client-side pages as well? @jinkang-0 

edit: i just implemented Link, but the underline color is wonky :<; any ideas on how to fix this? (also not a huge issue)
<img width="998" alt="Screen Shot 2023-11-21 at 10 28 33 AM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/98826c6c-e390-46d0-9b10-52986283d0e3">

Reply:
That'll be great, but just be wary of merge conflicts!